java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -165,6 +188,9 @@ private static Object[] genericCommands(BaseTransaction<?> transaction) {
                         false, // pexpireAt(genericKey1, 42, ExpireOptions.HAS_NO_EXPIRY)
                         -2L, // expiretime(genericKey1)
                         -2L, // pexpiretime(genericKey1)
+                        ascendingList, // sortReadOnly(genericKey3)
+                        descendingList, // sortReadOnly(genericKey3,
```

Comment:
```suggestion
                        descendingList, // sortReadOnly(genericKey3, DESC)
```
It's a comment - we can fudge it a little